specification/api/components/schemas/Letters.yml

```diff
@@ -0,0 +1,6 @@
+type: object
```

Comment:
Should this be LetterListResponse.yml or something like that?

For list responses we will also want to include [pagination links](https://jsonapi.org/format/#fetching-pagination), and at a minimum should include a `links.self` field [according to the spec](https://jsonapi.org/format/#fetching-resources-responses).

Presumably if we did include pagination we would also need the corresponding query parameters in the request.